src/App.jsx

```diff
@@ -14,70 +15,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [goodsItem, setValue] = useState('Jam');
+  const titleDefaultText = 'No goods selected';
+  const titleSelectedText = `${goodsItem} is selected`;
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {goodsItem ? titleSelectedText : titleDefaultText}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+        {goodsItem && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={(e) => {
+              setValue('');
+            }}
+          />
+        )}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(item => (
+            <tr
+              data-cy="Good"
+              key={item}
+              className={
+                goodsItem === item
```

Comment:
Create a variable inside map function to avoid code duplication

`const isGoodSelected === goodsItem === item`

And don't forget to add return statement inside map